modular_skyrat/modules/modsuit_pai/code/mod_pai.dm

```diff
@@ -0,0 +1,137 @@
+/**
+ * Simple proc to insert the pAI into the MODsuit.
+ *
+ * user - The person trying to put the pAI into the MODsuit.
+ * card - The pAI card we're slotting in the MODsuit.
+ */
+
+/obj/item/mod/control/proc/insert_pai(mob/user, obj/item/paicard/card)
+	if(mod_pai)
+		balloon_alert(user, "pAI already installed!")
+		return
+	if(!card.pai || !card.pai.mind)
+		balloon_alert(user, "pAI unresponsive!")
+		return
+	balloon_alert(user, "transferring to suit...")
+	if(!do_after(user, 5 SECONDS, target = src))
+		balloon_alert(user, "interrupted!")
+		return FALSE
+	if(!user.transferItemToLoc(card, src))
+		return
+
+	mod_pai = card.pai
+	balloon_alert(user, "pAI transferred to suit")
+	balloon_alert(mod_pai, "transferred to a suit")
```

Comment:
Can maybe make these visible alerts

Reply:
Hmmm, I could I suppose, yeah. I hadn't really touched that part too much, though, I guess.
I'd try it out on live with just balloon alerts first, and then we'll make it a visible message if it feels necessary.